Test/git-issues/git-issue-1665.dfy

```diff
@@ -0,0 +1,38 @@
+// RUN: %dafny /env:0 /dafnyVerify:0 /compile:0 /rprint:- "%s" >> "%t"
+// RUN: %diff "%s.expect" "%t"
```

Comment:
You can use the cross-platform command `%sed` like in this [file](https://github.com/dafny-lang/dafny/blob/master/Test/dafny0/ShowSnippets.dfy) and run dafny a second time with /rprint:filename.dfy.raw perhaps?

Reply:
I changed the script a little, is it good ?

Reply:
It is great !

Reply:
Should I resolve this conversation, or there is something more to be done here?